### PR TITLE
docs: updating readme and version for rebranding

### PR DESCRIPTION
**Pull request by ashin-czi**
#309

#### Reviewers
**Functional:** @seve 

**Readability:** @seve @karenliang0 

---

## Changes
- add
- remove
- modify
Updating Readme, __init__.py, and infoMenu wording for rebranding

### CODE REVIEW

File: README.md
Comment (ambrosejcarr) [inline README.md:9]:
```suggestion
CZ CELLxGENE Explorer (pronounced "cell-by-gene") is an interactive data explorer for single-cell transcriptomics datasets, such as those coming from the [Human Cell Atlas](https://humancellatlas.org). Leveraging modern web development techniques to enable fast visualizations of at least 1 million cells, we hope to enable biologists and computational researchers to explore their data.
```

File: README.md
Comment (ambrosejcarr) [inline README.md:11]:
```suggestion
Whether you need to visualize one thousand cells or one million, CELLxGENE Explorer helps you gain insight into your single-cell data.
```

Per comments elsewhere, we're trying to establish CELLxGENE as the brand family, so are eliminating uses of that name to refer to individual applications. Here the correct referent is "Explorer"

File: README.md
Comment (ambrosejcarr) [inline README.md:21]:
```suggestion
CZ CELLxGENE Explorer currently supports the following browsers:
```
```suggestion
CZ CELLxGENE currently supports the following browsers:
```

File: README.md
Comment (ambrosejcarr) [inline README.md:17]: Whoops, this link is outdated. Sneaking in this change:

```suggestion
[The CZ CELLxGENE documentation is your one-stop-shop for information about CELLxGENE](cellxgene.cziscience.com/docs)!
```

File: README.md
Comment (ambrosejcarr) [inline README.md:36]:
```suggestion
# Developing with CZ CELLxGENE Explorer
```

File: README.md
Comment (ambrosejcarr) [inline README.md:40]:
```suggestion
Please see our [contributing guide](https://github.com/chanzuckerberg/cellxgene-documentation/blob/main/contribute.md) and don't hesitate to open an issue or send a pull request to improve CZ CELLxGENE Explorer. Please see the [dev_docs](https://github.com/chanzuckerberg/single-cell-explorer/blob/main/dev_docs) for pull request suggestions, unit test details, local documentation preview, and other development specifics. 
```

File: README.md
Comment (ambrosejcarr) [inline README.md:51]:
```suggestion
Before extending CZ CELLxGENE Explorer, we encourage you to reach out to us with ideas or questions. It might be possible that an 
```

File: client/src/components/leftSidebar/infoMenu.tsx
Comment (ambrosejcarr) [inline client/src/components/leftSidebar/infoMenu.tsx:38]:
```suggestion
            text="About CELLxGENE"
```

@seve can you sanity check this change please?

File: server/__init__.py
Comment (ambrosejcarr) [inline server/__init__.py:8]:
```suggestion
display_version = "CELLxGENE v" + __version__
```

**Review by ambrosejcarr — APPROVED** [review]
LGTM pending a few changes, but will want Seve's feedback before merge.

File: client/src/components/leftSidebar/infoMenu.tsx
Comment (seve) [reply client/src/components/leftSidebar/infoMenu.tsx:38]: should it be CZ CELLxGENE?

File: server/__init__.py
Comment (seve) [reply server/__init__.py:8]: let's keep code unchanged 
```suggestion
display_version = "cellxgene v" + __version__
```

**Review by seve — CHANGES_REQUESTED** [review]
just a few comments

**Comment by ashin-czi** [comment]
Made the changes per the comments above. Still not sure if on `infoMenu.ts` it should be "About CELLxGENE" or "About CZ CELLxGENE" @ambrosejcarr

File: client/src/components/leftSidebar/infoMenu.tsx
Comment (ambrosejcarr) [reply client/src/components/leftSidebar/infoMenu.tsx]: CZ CELLxGENE Discover.

Comment (ashin-czi) [reply client/src/components/leftSidebar/infoMenu.tsx]: Updated!

**Comment by ashin-czi** [comment]
@ambrosejcarr Think this needs approval again for it to be merged

File: README.md
Comment (seve) [inline README.md:17]: Let's swap to a relative `/docs`

File: client/index_template.html
Comment (seve) [inline client/index_template.html:6]:
```suggestion
    <title>CELL&times;GENE | Explorer</title>
```

File: client/src/components/app.tsx
Comment (seve) [inline client/src/components/app.tsx:57]:
```suggestion
              <Helmet title="CELL&times;GENE | Explorer" />
```

**Review by seve — DISMISSED** [review]

**Comment by ashin-czi** [comment]
@ambrosejcarr @seve 

I think we're close to merging this, could I get a final review for this whenever you guys get the chance?

**Review by ambrosejcarr — APPROVED** [review]

**Comment by ashin-czi** [comment]
Ready for staging

**Pull request merged**
